app/distributions/schemas.py

```diff
@@ -0,0 +1,30 @@
+from datetime import datetime
+
+from ninja import Schema
+from schemas import TranslationsSchema
+
+
+class AttributionSchema(Schema):
+    id: str
+    name: str
+    name_translations: TranslationsSchema
+    description: str
+    description_translations: TranslationsSchema
+    provider_id: str
+
+
+class AttributionListSchema(Schema):
+    items: list[AttributionSchema]
```

Comment:
I wonder if this could directly be a list instead of being nested in items (but it's done like specified in apidog) cc @boecklic

Reply:
I wondered the same, assumed it would serve a certain end as it's used in every example. :sweat_smile: